app/controllers/coronavirus_form/data_export_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::DataExportController < ApplicationController
+  def show
+    render plain: produce_csv(usage_statistics)
+  end
+
+private
+
+  def usage_statistics
```

Comment:
I would be great if this could have it's own test, so if we need expand the formats to include other types, like json, we'd have confidence in the results being extracted.

Reply:
Will add a unit test for this method.